airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/manifest.yaml

```diff
@@ -0,0 +1,156 @@
+version: 0.83.0
```

Comment:
You could bump this to 1.0+ as well, so version in the poetry.lock matchies this.